config/default_quotas.yml

```diff
@@ -0,0 +1,3 @@
+- { id: c088f732-0a30-454b-aa6b-542ce1d67bfb, resource_type: VmCores,           value: 8   }
+- { id: 14fa6820-bf63-41d2-b35e-4a4dcefd1b15, resource_type: GithubRunnerCores, value: 150 }
```

Comment:
Can we assert that all `id` and `resource_type` values are unique? Alternatively, could there be multiple instances of the same `resource_type` with different IDs? Is there any scenario for that?

Reply:
In the future, we might have other fields to split resource types. For now they are unique, so I added a test case.